lab 2/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>My title</title>
+
+      <!-- font awesome -->
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+   </head>
+   <body>
+   <header>
+      <div class="collapsed-menu">
+         <div class="title menu-item">Navigation</div>
+         <div class="button menu-item"><i class="fa fa-bars" aria-hidden="true"></i></div>
+      </div>
+      <div class="menu">
+         <div class="menu-item"><a href='#'>psd websites</a></div>
```

Comment:
li

Reply:
Can i expand area by making a structure like this:
a > li.menu-item?

Reply:
I guess no because semantically it's not a correct html structure

Reply:
ok. Thanks.